test/test_torch.py

```diff
@@ -3772,6 +3772,31 @@ def get_probs(length, is_contiguous):
                 a_t, p_t = torch._multinomial_alias_setup(probs)
                 torch._multinomial_alias_draw(p_t.view(2, 2), a_t.view(2, 2))
 
+        for probs in [get_probs(4, True), torch.tensor([0.8, 0.2]), torch.tensor([0.7, 0.2, 0.1])]:
+            # Check how different the alias distribution and the original distribution are
+            n_samples = 50000
+            alias_table, prob_table = torch._multinomial_alias_setup(probs)
+            alias_samples = torch._multinomial_alias_draw(prob_table, alias_table, n_samples)
+            alias_dist = torch.unique(alias_samples, return_counts=True)[1].to(dtype=probs.dtype) / n_samples
+            rtol = 0.1 if IS_WINDOWS else 0.02
+            self.assertTrue(torch.allclose(alias_dist, probs, rtol=rtol, atol=0.0))
```

Comment:
Using `self.assertEqual` or `self.assertAlmostEqual` might make it easier to debug test failures in CI. Alternatively you could add a `msg` arg to `assertTrue` to print out the values / diffs.